From 6d4f4145613743adfd2cd5a58efb6872ba2d571f Mon Sep 17 00:00:00 2001
From: allen yan <yanwei@marvell.com>
Date: Mon, 24 Apr 2017 16:19:35 +0800
Subject: [PATCH 0961/1345] dts: a37xx: change CPU decode window mapping for
 PCIe

commit  303e919a6411ffae41fc7709766ca2b9949fcff4 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

- In order to support 4GB DRAM board, the CPU decode window
  address for PCIe has been changed to 0xd8200000 in u-boot,
  which is 2M byte aligned required by MMU.
- This patch align with u-boot that also set PCIe memory start
  address to 0xd8200000.

Change-Id: Ieb10fd1e30ff696ae254406cad4dda3234c42380
Signed-off-by: allen yan <yanwei@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/38838
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Victor Gu <xigu@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-37xx.dtsi |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
index 48184b5..257158a 100644
--- a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
@@ -529,8 +529,8 @@
 			msi-parent = <&pcie0>;
 			msi-controller;
 			clocks = <&sb_periph_clk 13>;
-			ranges = <0x82000000 0 0xe8000000   0 0xe8000000 0 0x1000000 /* Port 0 MEM */
-				  0x81000000 0 0xe9000000   0 0xe9000000 0 0x10000>; /* Port 0 IO*/
+			ranges = <0x82000000 0 0xd8200000   0 0xd8200000 0 0x1000000 /* Port 0 MEM */
+				  0x81000000 0 0xd9200000   0 0xd9200000 0 0x10000>; /* Port 0 IO*/
 			interrupt-map-mask = <0 0 0 7>;
 			interrupt-map = <0 0 0 1 &pcie_intc 0>,
 					<0 0 0 2 &pcie_intc 1>,
-- 
1.7.9.5

